baton.lua

```diff
@@ -75,6 +75,14 @@ function sourceFunction.button(button)
   end
 end
 
+function sourceFunction.axispair(controls)
```

Comment:
Same order as Konami Code, btw. I figure that's the most intuitive way to order this. :)